websockets/websockets.go

```diff
@@ -586,6 +613,55 @@ func (w *webSocket) send(msg goja.Value) {
 	}
 }
 
+// Ping sends a ping message over the websocket.
+func (w *webSocket) ping() {
+	deadline := time.Now().Add(writeWait)
+	pingID := strconv.Itoa(w.sendPings.counter)
+
+	data := []byte(pingID)
+
+	err := w.conn.WriteControl(websocket.PingMessage, data, deadline)
```

Comment:
In the previous( `k6/ws`) implementation all the methods did what they did (send/ping ) on the event loop more or less. This is not the case here `send` does not actually send the message anymore it queues it to be send. Given that `ping` also writes I feel like it should be *off* the main loop the same way the `send` is